__init__.py

```diff
@@ -502,15 +504,20 @@ def msg_jira_issue_created(body, project):
         summary = body["issue"]["fields"]["summary"]
         description = body["issue"]["fields"]["description"]
 
-        return f"[JIRA] {user} created issue {key} {summary}\n" f"{url}\n" + description
+        return f"""[JIRA] {user} created issue <a href="{url}">{key}</a>
+                   <br>
+                   <b>{summary}<b>
+                   <pre>{description}</pre>"""
 
     @staticmethod
     def msg_jira_issue_deleted(body, project):
         user = body["user"]["displayName"]
         key = body["issue"]["key"]
         summary = body["issue"]["fields"]["summary"]
 
-        return f"[JIRA] {user} deleted an issue {key} {summary}"
+        return f"""[JIRA] {user} deleted an issue {key}
```

Comment:
```suggestion
        return f"""[JIRA] {user} deleted issue {key}
```
analogous to `created`